### ISSUE TITLE
"make check" fails when configured with --disable-xml-context

### ISSUE THREAD

**Issue by hartwork** [libexpat/libexpat#146]
```console
# ./configure CFLAGS='-Wall -Wextra -std=c99 -pedantic' CXXFLAGS='-Wall -Wextra -std=c++98 -pedantic' CPPFLAGS=-DXML_UNICODE_WCHAR_T --host=i686-w64-mingw32 --disable-xml-context --disable-shared && make clean check
[..]
FAIL: runtests.exe
FAIL: runtestspp.exe
============================================================================
Testsuite summary for expat 2.2.4
============================================================================
# TOTAL: 2
# PASS:  0
# SKIP:  0
# XFAIL: 0
# FAIL:  2
# XPASS: 0
# ERROR: 0
============================================================================
See tests/test-suite.log
Please report to expat-bugs@libexpat.org
============================================================================
[..]

# wine tests/runtests.exe 
Expat version: expat_2.2.4
ERROR: Parsing failed even at max reallocation count
99%: Checks: 328, Failed: 1
```

**Comment by hartwork** [comment]
@RMJ10 if you get a chance to have a look, that would be awesome.

**Comment by hartwork** [comment]
Update: Turned out to fail independently of `XML_UNICODE_WCHAR_T`.

**Comment by hartwork** [comment]
Fixed now, found bc6468d23073d133ef1b439b5049c7f609595087 to be the troublemaker using `git bisect`:

```
cd ..  # project root
git bisect start
git bisect bad master
git bisect good R_2_2_4
git bisect run sh -c "cd expat && ./buildconf.sh && ./configure CFLAGS='-Wall -Wextra -std=c99 -pedantic' CXXFLAGS='-Wall -Wextra -std=c++98 -pedantic' --disable-xml-context --disable-shared && make clean check"
```
